tests/unit/admin/test-class-wc-payments-admin.php

```diff
@@ -108,13 +108,13 @@ public function test_it_does_not_render_payments_badge_if_stripe_is_connected()
 		$this->assertArrayNotHasKey( 'wc-admin&path=/payments/connect', $item_names_by_urls );
 	}
 
-	public function test_it_renders_payments_badge_if_activation_date_is_older_than_7_days_and_stripe_is_not_connected() {
+	public function test_it_renders_payments_badge_if_activation_date_is_older_than_3_days_and_stripe_is_not_connected() {
```

Comment:
The test below this one `test_it_does_not_render_payments_badge_if_activation_date_is_less_than_7_days` will need fixing to, it is failing at the moment

Reply:
Thanks, @louwie17 - completely missed that one.  Ready for re-review.